agent/runtime-attach/build.gradle.kts

```diff
@@ -0,0 +1,11 @@
+plugins {
+  id("ai.java-conventions")
+}
+
+dependencies {
+  implementation("io.opentelemetry.contrib:opentelemetry-runtime-attach:1.14.0-alpha")
```

Comment:
let's add `otelContribAlphaVersion` similar to `otelInstrumentationAlphaVersion`:

(we probably need to keep separate version constants for now since they are released from separate repos not always at the same time)

https://github.com/microsoft/ApplicationInsights-Java/blob/7eb023e32f5ec4987c09f618f326ff92baf41815/dependencyManagement/build.gradle.kts#L17

Reply:
Good point. Code updated. Please check [this](https://github.com/microsoft/ApplicationInsights-Java/blob/runtime-attach/dependencyManagement/build.gradle.kts) and [this](https://github.com/microsoft/ApplicationInsights-Java/blob/runtime-attach/agent/runtime-attach/build.gradle.kts).